### ISSUE TITLE
Dashboard improvement: show response times in a graph

### ISSUE THREAD

**Issue by JorritSalverda** [Netflix/Hystrix#1025]
The current graphs in the dashboard show the request volume plotted against time. Plotting the response times as well either in a separate graph or in the same graph would be very useful.

Best would be to graph the response times for the different percentiles, as it provides much more insight whether a large percentage is suffering from degraded response times or just a small portion.

As the default values for bucket length in ms are much longer than those for nr of requests the graph might be too rough though.

**Comment by mattrjacobs** [comment]
Yes, this seems like a good idea.  It's pretty unlikely this makes it to the top of my priority list, however.  See #1033 for a discussion on the dashboard roadmap.

**Comment by mukteshkrmishra** [comment]
@mattrjacobs : I can contribute as well. Please let me know in case you need it.

**Comment by mattrjacobs** [comment]
I'd be happy to review any contributions for the hystrix-dashboard (this one included).  I won't have time to work on them, but can certainly review and answer questions.  

Is there anything I can do from an issue management perspective that would make it easier for contributors to find this type of issue?  I know RxJava uses a 'PRs Welcome' tag for features that are valid but not worked on by the core team.
